subtests/docker_cli/negativeusage/negativeusage.py

```diff
@@ -151,6 +151,7 @@ def do_substitutions(self):
                                                  xcpt.message))
             else:
                 value = ''
+            # pylint: disable=E0012,R0204
             if key == 'subarg':  # This is a CSV option
                 if value.strip() == '':
                     value = []
```

Comment:
R0204:156,20: Base.do_substitutions: Redefinition of value type from str to list